tasks/sep/zora-002-fp-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,52 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction. For each contract listed in the state diff, please verify that no contracts or state
+changes shown in the Tenderly diff are missing from this document.
+
+Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no
+  unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the
+  Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the
+  addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage
+  layout links provided.
+
+## State Overrides
+
+### SystemOwnerSafe (0x1Eb2fFc903729a0F03966B917003800b145F56E2)
```

Comment:
We can refer to `NESTED-VALIDATION.md` for the full set of state overrides since these are constant